docs/source/ui/pages/table.md

```diff
@@ -0,0 +1,42 @@
+## Table
+
+### Overview
+
+The table helps to explore and compare training runs results, logged hyper-parameters, system info, etc. As well as to apply quick actions (hide, delete, archive, etc.) over a batch of selected items within a few clicks.
+
+### Delete a batch of items
+
+To delete a batch of runs select rows and click on the `Delete` button, which appears at the top of the table.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC; height: 200px" alt="Delete a batch of items" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/delete_a_batch_of_items.png">
+
+.. note::
+In-progress runs can't be deleted. Wait until the process is finished or terminate the process to be able to delete them.
```

Comment:
... can't be deleted. Please wait until the run is finished or terminate it to be able to delete the run.